repos/system_upgrade/common/actors/cephvolumescan/libraries/cephvolumescan.py

```diff
@@ -63,7 +63,11 @@ def encrypted_osds_list():
         output = get_ceph_lvm_list()
         if output is not None:
             try:
-                result = [output[key][0]['lv_uuid'] for key in output if output[key][0]['tags']['ceph.encrypted']]
+                result = []
+                for key in output:
```

Comment:
`result=[]` at L66 looks like redefinition from L61.
Linter is unhappy at the moment about too many nested blocks, maybe something like this will do?

```
if not output:
    return []
 
for key, elements in output.items():
    for element in elements:
        try:
            if element['tags']['ceph.encrypted']:
                result.append(element['lv_uuid'])
        except KeyError:
            api.current_logger().warning('ceph-osd is installed but no encrypted osd has been found')
            break
return result         
```